src/content.js

```diff
@@ -465,6 +466,26 @@ function addTitleToEmojis() {
 	}
 }
 
+function shortenVisibleUrls() {
+	for (const a of select.all('a[href]')) {
+		// Don't change if it was already customized
+		// .href automatically adds a / to naked origins
+		// so that needs to be tested too
+		if (a.href !== a.textContent && a.href !== `${a.textContent}/`) {
+			continue;
+		}
+
+		const shortened = shortenUrl(a.href);
+
+		// Don't touch the dom if there's nothing to change
+		if (shortened === a.href) {
```

Comment:
They're the same with the exception that `.href` can contain an extra blackslash. I'll change it to `.textContent`